Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/OverworldManager.cs

```diff
@@ -185,7 +187,50 @@ void _StormshapersPlayer_OnCardRemoved(CardData card, int index)
 
 	}
 
+	void _PointsSystem_CurrencyChanged(int val)
+	{
+		_OverworldUI._ArmouryUI_OnCurrencyChanged(val, _GameStateHolder._ActivePlayer);
```

Comment:
It could. I didn't think you wanted gamestateholder being in OverworldUI so I just used put it in manager instead.